Jep334Tests_0 failed in x86-64_linux_vt_standard

Failure link
https://openj9-jenkins.osuosl.org/job/Test_openjdknext_j9_sanity.functional_x86-64_linux_vt_standard_Personal_testList_0/17/tapResults/

Failure output (captured from console output)
---------------------------------------------
There are 2 types of failures. 
1. IllegalAccessException
```
java.lang.IllegalAccessException: access violation: class java.util.Objects$1, from class org.openj9.test.java_lang.Test_Class (unnamed module @b34edd1)
	at java.base/java.lang.invoke.MethodHandles$Lookup.makeAccessException(MethodHandles.java:2856)
	at java.base/java.lang.invoke.MethodHandles$Lookup.accessClass(MethodHandles.java:2934)
	at java.base/java.lang.invoke.MethodHandles$Lookup.findClass(MethodHandles.java:2793)
	at java.base/java.lang.constant.ClassDescImpl.resolveConstantDesc(ClassDescImpl.java:79)
	at java.base/java.lang.constant.ClassDescImpl.resolveConstantDesc(ClassDescImpl.java:40)
	at org.openj9.test.java_lang.Test_Class.describeConstableTestGeneral(Test_Class.java:121)
	at org.openj9.test.java_lang.Test_Class.testClassDescribeConstable(Test_Class.java:108)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:76)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:52)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
```

2 AssertionError:
```
java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertTrue(Assert.java:44)
	at org.testng.Assert.assertTrue(Assert.java:54)
	at org.openj9.test.java_lang.Test_Class.testClassDescriptorString(Test_Class.java:83)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```

--- a/test/functional/Java12andUp/src/org/openj9/test/java_lang/Test_Class.java
+++ b/test/functional/Java12andUp/src/org/openj9/test/java_lang/Test_Class.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018, 2020 IBM Corp. and others
+ * Copyright (c) 2018, 2021 IBM Corp. and others
  *
  * This program and the accompanying materials are made available under
  * the terms of the Eclipse Public License 2.0 which accompanies this
@@ -56,12 +56,12 @@ public class Test_Class {
 	};
 
 	/* descriptor strings */
-	String objectResult = "Ljava/lang/Object;";
+	String objectResult = "Ljava/lang/Thread;";
 	String arrayResult = "[D";
 	String arrayResult2 = "[[[D";
 
 	/* test classes */
-	Class<?> objectTest = new Object().getClass();
+	Class<?> objectTest = new Thread().getClass();
 	Class<?> arrayTest = new double[0].getClass();
 	Class<?> arrayTest2 = new double[0][][].getClass();
 	Runnable lambdaExp = () -> System.out.println("test");

